mbta-card.js

```diff
@@ -46,6 +46,10 @@ class MBTACard extends HTMLElement {
 
         span.blue {
             background-color: #003da5;
+        }
+
+        span.commuter {
+            background-color: #80276c;
```

Comment:
the official hex code for the mbta commuter rail. NICE!

Reply:
Hah, I stare at it every day, I'd go nuts if it was off!